core/src/hir/attrs.rs

```diff
@@ -750,6 +750,8 @@ pub struct BackendAttrSupport {
     pub utf8_strings: bool,
     /// Whether the language uses UTF-16 strings
     pub utf16_strings: bool,
+    /// Whether the language supports returning or using slices with 'static lifetimes.
```

Comment:
returning should always be fine, they can be treated like any other borrow that doesn't have to be freed. the unsupported bit is *creating* a static slice in another language (without leaking), which is unsupported in JS (and Dart).